Web_Development/FrontEnd/Htmlformtogooglesheet/Readme.md

```diff
@@ -0,0 +1,256 @@
+          <span style="color: #f92672">.</span><span style="color: #f8f8f2">then((response)</span> <span style="color: #f92672">=&gt;</span> <span style="color: #f8f8f2">alert(</span><span style="color: #e6db74">&quot;Thanks for fiiling the form&quot;</span><span style="color: #f8f8f2">))</span>
+          <span style="color: #f92672">.</span><span style="color: #f8f8f2">catch((error)</span> <span style="color: #f92672">=&gt;</span> <span style="color: #f8f8f2">console</span><span style="color: #f92672">.</span><span style="color: #f8f8f2">error(</span><span style="color: #e6db74">&quot;Error!&quot;</span><span style="color: #f8f8f2">,</span> <span style="color: #f8f8f2">error</span><span style="color: #f92672">.</span><span style="color: #f8f8f2">message));</span>
+      <span style="color: #f8f8f2">});</span>
+    <span style="color: #f92672">&lt;/</span><span style="color: #f8f8f2">script</span><span style="color: #f92672">&gt;</span>
+    <span style="color: #f92672">&lt;</span><span style="color: #f8f8f2">script</span> <span style="color: #f8f8f2">src</span><span style="color: #f92672">=</span><span style="color: #e6db74">&quot;https://maxcdn.bootstrapcdn.com/bootstrap/4.1.1/js/bootstrap.min.js&quot;</span><span style="color: #f92672">&gt;&lt;/</span><span style="color: #f8f8f2">script</span><span style="color: #f92672">&gt;</span>
+    <span style="color: #f92672">&lt;</span><span style="color: #f8f8f2">script</span> <span style="color: #f8f8f2">src</span><span style="color: #f92672">=</span><span style="color: #e6db74">&quot;https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js&quot;</span><span style="color: #f92672">&gt;&lt;/</span><span style="color: #f8f8f2">script</span><span style="color: #f92672">&gt;</span>
+  <span style="color: #f92672">&lt;/</span><span style="color: #f8f8f2">body</span><span style="color: #f92672">&gt;</span>
+<span style="color: #f92672">&lt;/</span><span style="color: #f8f8f2">html</span><span style="color: #f92672">&gt;</span>
+</pre></div>
+
+<hr>
+
+**Which basically looks like this :-**
+<br>
+
+**Front Page UI**
+
+![Front Page UI](./FrontformUI.png)
+
+# Javasript code
+
+The below mention code which links the form with the html forms
+<br>
+
+**Here is the javascript code**
+<div style="background: #272822; overflow:auto;width:auto;border:solid gray;border-width:.1em .1em .1em .8em;padding:.2em .6em;"><pre style="margin: 0; line-height: 125%"><span style="color: #f8f8f2">var</span> <span style="color: #f8f8f2">GooglesheetName</span> <span style="color: #f92672">=</span> <span style="color: #e6db74">&quot;Sheet1&quot;</span><span style="color: #f8f8f2">;</span>
```

Comment:
Here also, use code blocks with syntax.

Reply:
no its not a code it is for readme.d file it will be back at the time of showing the readme file the above is to codeblock the syntax of readme